test/gtest/uct/test_event.cc

```diff
@@ -164,14 +153,24 @@ void test_uct_event_fd::test_recv_am(bool signaled)
     free(recv_buffer);
 }
 
-UCS_TEST_P(test_uct_event_fd, am)
+UCS_TEST_SKIP_COND_P(test_uct_event_fd, am,
+                     !check_caps(UCT_IFACE_FLAG_EVENT_RECV |
+                                 UCT_IFACE_FLAG_CB_SYNC    |
+                                 UCT_IFACE_FLAG_AM_BCOPY))
 {
-    test_recv_am(false);
+    m_arm_flags  = UCT_EVENT_RECV;
```

Comment:
why not pass arm_flags+send_flags as parameters to test_recv_am()?

Reply:
done